Assets/Python/GreatPeople.py

```diff
@@ -516,7 +568,7 @@ def setup():
 		iClassical,
 		"Mago", # 4th BC
 		"Hasdrubal Clitomachus", # 2nd BC
-		"Abba", # 3rd 
+		"Abba", # 3rd
```

Comment:
Now a Swedish great person? :D

Reply:
Haha, I have no idea who that is

Reply:
Me neither, I think the name comes from the original civ specific GP name mod.